RijsMuseum/app/src/main/res/layout/fragment_collection_details.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="#434343"
```

Comment:
Don't use colors like that. Always define them in colors.xml or use the android default ones.